OJP/OJP_JourneySupport.xsd

```diff
@@ -102,7 +102,11 @@
 					<xs:documentation>Contains the Vehicle reference of the vehicle. In Transmodel this may be the VEHICLE Id.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:element ref="OperatorRef" minOccurs="0"/>
+			<xs:element name="OperatorRefs" type="OperatorsRefsStructure" minOccurs="0">
```

Comment:
I see that it's consistently everywhere the type *OperatorsRefs..* - but name *OperatorRefs*. My preferred choice would be *OperatorRefs...*.

Reply:
adapted